patches/minecraft/net/minecraft/entity/passive/EntityHorse.java.patch

```diff
@@ -1,22 +1,52 @@
 --- ../src-base/minecraft/net/minecraft/entity/passive/EntityHorse.java
 +++ ../src-work/minecraft/net/minecraft/entity/passive/EntityHorse.java
-@@ -441,6 +441,7 @@
+@@ -400,12 +400,13 @@
+                 }
+             }
+ 
+-            IBlockState iblockstate = this.field_70170_p.func_180495_p(new BlockPos(this.field_70165_t, this.field_70163_u - 0.2D - (double)this.field_70126_B, this.field_70161_v));
++            BlockPos pos = new BlockPos(this.field_70165_t, this.field_70163_u - 0.2D - (double)this.field_70126_B, this.field_70161_v);
++            IBlockState iblockstate = this.field_70170_p.func_180495_p(pos);
+             Block block = iblockstate.func_177230_c();
```

Comment:
same here. splitting the pos out saves you a couple nanoseconds and increases the patch size